etc/behat/services/contexts.xml

```diff
@@ -26,6 +26,9 @@
         <parameter key="sylius.behat.context.setup.tax.class">Sylius\Behat\Context\Setup\TaxContext</parameter>
         <parameter key="sylius.behat.context.setup.order.class">Sylius\Behat\Context\Setup\OrderContext</parameter>
         <parameter key="sylius.behat.context.setup.theme.class">Sylius\Behat\Context\Setup\ThemeContext</parameter>
+        <parameter key="sylius.behat.context.setup.customer.class">Sylius\Behat\Context\Setup\CustomerContext</parameter>
+        <parameter key="sylius.behat.context.setup.addressing.class">Sylius\Behat\Context\Setup\AddressingContext</parameter>
+        <parameter key="sylius.behat.context.setup.taxonomy.class">Sylius\Behat\Context\Setup\TaxonomyContext</parameter>
```

Comment:
Won't it be better to rename this context and parameters to taxon? According to #3773 taxon should be more appropriate.